remind_class.py

```diff
@@ -0,0 +1,77 @@
+import email, smtplib, ssl
+from email.message import EmailMessage
+import time 
+from datetime import datetime
+
+################################################################################
+def look_up_gateway():
+    gate_way = {'AirFire Mobile': "number@sms.airfiremobile.com", 'Aio Wireless': "number@mms.aiowireless.net", 
+    'Alaska Communications': "number@msg.acsalaska.com", 'Alltel (Allied Wireless)' : "number@mms.alltelwireless.com", 'Verizon': "number@vzwpix.com.",
+    "Assurance Wireless": "number@vmobl.com", "AT&T": "number@mms.att.net", "BellSouth": "number@bellsouth.cl", "Bluegrass Cellular": "number@mms.myblueworks.com",
+    'Bluesky Communications': "number@psms.bluesky.as", 'Boost mobile': "number@myboostmobile.com", 'CellCom': "number@cellcom.quiktxt.com", 'Cellular South':"number@csouth1.com",
+    'Centennial Wireless': "number@cwemail.com", 'Chariton Valley Wireless': "number@sms.cvalley.net", 'Chat Mobility': "number@mail.msgsender.com",
+    'Cincinnati Bell': "number@mms.gocbw.com", 'Cleartalk': "number@sms.cleartalk.us", 'Cricket': "number@mms.cricketwireless.net", 'C Spire Wireless': "number@cspire1.com", 
+    'Edge Wireless': "number@sms.edgewireless.com", 'Element Mobile': "number@SMS.elementmobile.net", 'General Communications Inc.': "number@mobile.gci.net", 
+    'Golden State Cellular': "number@gscsms.com", 'GreatCall': "number@vtxt.com", 'MetroPCS': "number@mymetropcs.com", 'Pocket Wireless': "number@sms.pocket.com", 'Red Pocket Mobile':"number@vtext.com"}   
+    key = input('What carrier do you use: ')
+    if key in gate_way:
+      return gate_way[key]
+    else:
+      print("Sorry, carrier not found.")
+
+def email_alert(subject,body, to):
```

Comment:
You should add `docstrings` to each function. Docstrings are an explanation of the function, what the input arguments are, and what is returned by the function. This helps others understand your code.

Here is an example:
```
def add_two_numbers(a, b):
    """Adds two numbers and returns the result.

    Args:
        a (float): the first number in the sum.
        b (float): the second number in the sum.

    Returns:
        (float): the sum of numbers a and b.

```

https://www.pythontutorial.net/python-basics/python-function-docstrings/


Not all functions return values though. So you can explain what the function does in the first few lines of the description and skip the "Returns" part.